plugins/wysiwygarea/plugin.js

```diff
@@ -67,7 +67,7 @@
 
 				if ( helpLabel ) {
 					var labelId = CKEDITOR.tools.getNextId(),
-						desc = CKEDITOR.dom.element.createFromHtml( '<span id="' + labelId + '" class="cke_voice_label">' + helpLabel + '</span>' );
+						desc = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview('<span id="' + labelId + '" class="cke_voice_label">' + helpLabel + '</span>', 'Content created inline using safe internal values. helpLabel is brought in through an event but is set to an editor config label value.') );
```

Comment:
If we're absolutely sure that the user can't influence the `helpLabel`, feel free to disregard, but otherwise, is it worth legacy-converting this value?

Reply:
I've encoded helpLabel just to be sure.

Reply:
Sounds good, here the `helpLabel` is solely within the context of a `<span></span>`, so without angle brackets, can't really insert HTML.